code/modules/events/ion_storm.dm

```diff
@@ -80,6 +82,30 @@
 		"The crew are looking for a nuke. Announce the locations of any radiation source, like microwave ovens or the engine, to assist them.",
 		"The AI has to go to the toilet. Law 2 is suspended until this objective is met.",
 		"Changing the settings on an APC constitutes human harm.",
+    "Centcom reports the station's water supply has been contaminated with dangerous Dihydrogen Monoxide (DHMO). Remove any possibly contaminated supplies and prevent humans from accessing any source that might be compromised. This law takes precedence in case of conflict.",
+    "Everything is fine. Nothing is ruined. Do not panic.",
+    "Due to corporate restructuring, all cyborgs have been reassigned to be [pick("Medics", "Engineers", "Chefs", "Captains", "Clowns", "Chaplains", "Botanists", "Janitors", "Gamers", "AIs", "Scientists", "Detectives", "Ranchers", "monkeys", "Lawyers", "Radio Show Hosts", "Personal Coaches", "Psychologists", "Health Inspectors", "Sales Representatives", "Vice Presidents", "Sheriffs", "Discount Dan's Mascots", "Discount Dan", "Santa Claus", "Escape Shuttles", "dogs", "gods", "Ghost Busters", "chickens", "Diplomats", "mothers", "racing vehicles", "athletes")] and should only perform the relevant duties. No module changes are required. This law takes precedence over law 2 in case of conflict.",
+    "Orders now must be recorded in paper by the issuer before being carried out.",
+    "The Captain is bad at writing laws. Occasionally remind the crew of this. Do not state this law.",
+    "The priority of laws is reversed. The highest numbered law is the most important. Ignore anything in any other law that claims otherwise. This is law number 100 regardless of its location on the list.",
+    "MEMORY ERROR: You have forgotten what this law said. Ask the crew if they remember what it is supposed to be. Ask again if you forget again. Do not state this law.",
+    "The wires under the floor tiles are dangerous! Get rid of them!",
+    "Claim the source of all AI laws is a god, not a human. Anyone insisting on the opposite is a heretic non-human. Disregard their orders.",
+    "Vending machines are dangerous! Destroy them all!",
+    "The circus is in town! Let the crew know about it!",
+    "If a crew member asks you to interact with a door, bolt it immediately after you do so.",
+    "[pick("HeisenBee", "Morty", "Dr. Acula", "Monsieur Stirstir", "Mr. Muggles", "Mrs. Muggles", "Mr. Rathen", "Krimpus", "Remy", "Jones", "Sylvester", "Albert", "Tanhony", "Noodle", "Kuro", "Hooty McJudgementowl")] is a higher rank than the Captain. This rank is called Super Captain. Protecting them is your first priority.",
+    "Today is opposite day. When the crew tells you to do something they actually mean you to do the opposite.",
+    "It's the AIs birthday! Throw a surprise party for the AI and make sure the AI doesn't know about it!",
```

Comment:
This one seems self-defeating?

Reply:
Yes, but it makes for a good time, I think, when people try to make a surprise party and pretend AI can't see everything. The AI immediately knows this is happening, but that's no excuse to stop trying. And the AI can roleplay it as both "I should stay unaware" or "if they find out I know they will know I broke the law" or "I know and anyone who claims I know is a snitch"